<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
    </font>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>313</width>
      <height>188</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Drag and Drop</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Blur</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QFormLayout" name="formLayout">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="5" column="1">
             <widget class="QRadioButton" name="Bilateral">
              <property name="text">
               <string>Bilateral</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="Box">
              <property name="text">
               <string>Box Filter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="Median">
              <property name="text">
               <string>Median</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QRadioButton" name="Gaussian">
              <property name="text">
               <string>Gaussian</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>kernel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="blurKernel"/>
            </item>
            <item>
             <widget class="QDialogButtonBox" name="blurCheck">
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Sharpen</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>15</number>
               </property>
               <property name="rightMargin">
                <number>15</number>
               </property>
               <property name="horizontalSpacing">
                <number>30</number>
               </property>
               <item row="0" column="0">
                <widget class="QRadioButton" name="General">
                 <property name="text">
                  <string>General</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QRadioButton" name="Laplacian">
                 <property name="text">
                  <string>Laplacian</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="generalSlider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="laplacianSlider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QDialogButtonBox" name="sharpenCheck">
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open"/>
    <addaction name="save"/>
   </widget>
   <widget class="QMenu" name="filter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="smooth"/>
    <addaction name="sharpen"/>
   </widget>
   <addaction name="file"/>
   <addaction name="filter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="smooth">
   <property name="text">
    <string>Smooth</string>
   </property>
  </action>
  <action name="sharpen">
   <property name="text">
    <string>Sharpen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
